templates/security/esm.html

```diff
@@ -8,318 +8,555 @@
-      <p>ESM is available through an Ubuntu Advantage for Infrastructure subscription for physical servers, virtual machines, containers and desktops, and is free for personal use. ESM is also included in the Ubuntu Pro premium images that are optimised for <a href="/cloud/public-cloud">the public cloud</a>, and these also provide security maintenance for high and critical CVEs for the entire collection of software packages shipped with Ubuntu.</p>
-      <p>
-        <a href="/pro" class="p-button--positive">Get started with Ubuntu Advantage</a>
-        <a href="/security/esm#get-in-touch" class="p-button">Contact us</a  >
-        </p>
+<section class="p-strip--image is-deep" style="background-image:url('https://assets.ubuntu.com/v1/18717572-05_suru2_light_glow_8K_edited.png'); background-position: 70% bottom;">
+  <div class="u-fixed-width p-strip is-shallow u-no-padding--top">
+      <h1 class="p-heading--2 u-no-margin--bottom"><strong>Extended Security Maintenance (ESM)</strong></h1>
+      <h2 class="u-no-padding--top u-text--muted">Extra security patching for Ubuntu LTS, now included in Ubuntu Pro</h2>
+  </div>
+</section>
+<section class="p-strip is-deep u-no-padding--bottom">
+  <div class="p-strip u-no-padding--top">
+    <div class="row">
+      <div class="col-6">
+        <h2>10 year security coverage</h2>
       </div>
-      <div class="col-4 u-vertically-center u-hide--medium u-hide--small u-align--center">
-        {{
-          image(
-          url="https://assets.ubuntu.com/v1/92a095cb-Shield-greentick.svg",
-          alt="",
-          height="226",
-          width="190",
-          hi_def=True,
-          loading="auto",
-          ) | safe
-        }}
+      <div class="col-6">
+        <p>Security maintenance for the entire collection of software packages shipped with Ubuntu.</p>
+        <p>ESM enables continuous vulnerability management for critical, high and medium CVEs.</p>
+        <a href="/security/esm#get-in-touch" class="p-button--positive js-invoke-modal">Contact us</a>
       </div>
     </div>
-  </section>
+  </div>
+</section>
 
-  <div class="p-strip is-shallow">
-    <div class="u-fixed-width">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          <h4 class="p-heading-icon__title u-no-max-width u-no-margin--bottom">Extended Security Maintenance for Ubuntu 16.04 LTS is available from April 2021 until 2026. <a href="/blog/ubuntu-16-04-lts-transitions-to-extended-security-maintenance-esm">Learn more&nbsp;&rsaquo;</a></h4>
-        </div>
-      </div>
+<section class="p-strip is-deep u-no-padding--top">
+  <div class="row">
+    <div class="col-small-1 col-medium-1 col-1">
```

Comment:
Is there a reason for not using the vanilla logo pattern here?

Reply:
Yes, because in responsive mode, it will show a huge image taking the entire width of the screen